returnn/torch/engine.py

```diff
@@ -366,6 +366,10 @@ def train_epoch(self):
         extern_data = None
         num_seqs = None
         last_seq_idx = 0
+
+        ep_data_len = 0
+        ep_data_space = 0
```

Comment:
You should better make this a `NumberDict` and keep separate entries per data. At least I also want to know about this. Also, you cannot just sum them up as you do now, as the length might have totally different scales, but anyway different implications on memory usage.